### PR TITLE
MM-27208 MM-T1682 Add E2E test for joining an archived channel via a permalink

### PR DESCRIPTION
**Pull request by hmhealey**
#### Ticket Link
https://mattermost.atlassian.net/browse/MM-27208
https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin%3Acom.kanoah.test-manager__main-project-page#!/testCase/MM-T1682

### CODE REVIEW

**Review by Willyfrog — APPROVED** [review]
LGTM

File: e2e/cypress/integration/archived_channel/join_archived_channel_spec.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
```
Comment (srkgupta) [inline e2e/cypress/integration/archived_channel/join_archived_channel_spec.js:9]: Suggested group:
`// Group: @channel`

File: e2e/cypress/integration/archived_channel/join_archived_channel_spec.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    let testUser;
+
+    before(() => {
+        cy.apiInitSetup().then(({team, user}) => {
+            testTeam = team;
+            testUser = user;
+        });
+    });
+
+    it('MM-T1682 joining an archived public channel via a permalink', () => {
```
Comment (srkgupta) [inline e2e/cypress/integration/archived_channel/join_archived_channel_spec.js:23]: Suggested change:
`MM-T1682 Join an archived public channel by selecting a permalink to one of its posts`

**Review by srkgupta — APPROVED** [review]
Hi @hmhealey 
Overall the test looks great and is working fine. Just a couple of minor suggestions. Please change the same before you merge the PR. Thanks.

**Review by jgilliam17 — APPROVED** [review]
Thank you @hmhealey
Looks good.

- Verified test case steps, details and links

**Pull request merged**